src/Integration.Vsix/CFamily/PortedFromJava/Protocol.cs

```diff
@@ -50,6 +50,8 @@ public static void Write(BinaryWriter writer, Request request)
             Write(writer, request.Macros);
             WriteUTF(writer, request.TargetTriple);
             WriteUTF(writer, request.File);
+            WriteUTF(writer, /* pchDir */ "");
```

Comment:
@Godin what are these new fields? I couldn't find anything in your release notes that relate to this.

Reply:
@rita-g-sonarsource you're looking on the wrong commits - here is commits from version 6.12 to 6.13:
https://github.com/SonarSource/sonar-cpp/compare/6.12.0.20255...6.13.0.22261

And these fields related to https://jira.sonarsource.com/browse/CPP-1602 , which is relevant only for SonarQube and SonarClound, should be empty (`""`) in case of SonarLint.